rest/user_api_test.go

```diff
@@ -1345,57 +1345,182 @@ func TestUserXattrAvoidRevisionIDGeneration(t *testing.T) {
 }
 
 func TestGetUserCollectionAccess(t *testing.T) {
+	numCollections := 2
+	base.RequireNumTestDataStores(t, numCollections)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
 
-	rt := NewRestTester(t, nil)
-	defer rt.Close()
+	testBucket := base.GetPersistentTestBucket(t)
+	defer testBucket.Close()
+	scopesConfig := GetCollectionsConfig(t, testBucket, 2)
+	dataStoreNames := GetDataStoreNamesFromScopesConfig(scopesConfig)
+
+	scope1Name, collection1Name := dataStoreNames[0].ScopeName(), dataStoreNames[0].CollectionName()
+	collection2Name := dataStoreNames[1].CollectionName()
+	scopesConfig[scope1Name].Collections[collection1Name] = CollectionConfig{}
 
+	rtConfig := &RestTesterConfig{
+		CustomTestBucket: testBucket,
+		SyncFn:           `function(doc) {channel(doc.channel); access(doc.accessUser, doc.accessChannel);}`,
+		DatabaseConfig: &DatabaseConfig{
+			DbConfig: DbConfig{
+				Scopes:           scopesConfig,
+				NumIndexReplicas: base.UintPtr(0),
+				AutoImport:       base.TestUseXattrs(),
+			},
+		},
+	}
+
+	rt := NewRestTesterMultipleCollections(t, rtConfig, 2)
+	defer rt.Close()
```

Comment:
This could be changed to:

```
        rtConfig := &RestTesterConfig{
                CustomTestBucket: testBucket,
                SyncFn:           `function(doc) {channel(doc.channel); access(doc.accessUser, doc.accessChannel);}`,
        }

        rt := NewRestTesterMultipleCollections(t, rtConfig, 2)
        defer rt.Close()

        scope1Name := rt.GetDbCollections()[0].ScopeName
        collection1Name := rt.GetDbCollections()[0].Name
        collection2Name := rt.GetDbCollections()[1].Name
```